spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+feature 'Likes' do
+# As a user
+# So that I can show my appreciation for a picture
+# I would like to 'like' a picture
+  context 'when signed in' do
+    let(:user){User.create(name: 'Betty Draper', email: 'betty@drapers.com',
+                            username: 'bettydraper', password: 'pigeons123',
+                            password_confirmation: 'pigeons123')}
+    let(:user2){User.create(name: 'Don Draper', email: 'don@drapers.com',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.